wallet/src/repository/wallet/tests/mod.rs

```diff
@@ -577,10 +552,16 @@ fn test_create_vtt_does_not_spend_utxos() {
 
     assert_eq!(1, wallet.balance().unwrap().confirmed.available);
     assert_eq!(0, wallet.balance().unwrap().confirmed.locked);
-    assert_eq!(1, db.get::<_, u64>(&keys::account_balance(0)).unwrap());
```

Comment:
This was the only type mismatch between `db.put` and `db.get`, and this test did not fail because bincode serializes structs by concatenating all the fields, and the first field was a `u64` with value 1.